variables.tf

```diff
@@ -221,6 +221,46 @@ variable "jira_db_master_password" {
   }
 }
 
+variable "jira_shared_home_size" {
+  description = "Storage size for Jira shared home"
+  type        = string
+  default     = "10Gi"
+}
+
+variable "jira_nfs_requests_cpu" {
+  description = "The minimum CPU compute to request for the NFS instance"
+  type        = string
+  default     = "0.25"
```

Comment:
We may need to makes the default value for nfs resources consistent with Bitbucket. Unless we have a reason why those are different.

Reply:
Which property has inconsistent value? I checked and they all same